http2_general/test_h2_supported_methods.py

```diff
@@ -0,0 +1,70 @@
+from test_suite import tester
+
+
+class TestHttp2Methods(tester.TempestaTest):
```

Comment:
Let's move these tests to `test_h2_headers.py`. Method is pseudo header in http2.

Reply:
This test doesn't covers #2346 at all. In the test we have to test dynamic table, not regular h2 parser. Also the test doesn't covers the main case where `GET` can be matched to `GETASD` or to `GFD`.

@RomanBelozerov @WitalyAnisimov  I'm suggesting to move the test to `test_h2_hpack.py`. In this file there is class `TestHpackUnknownMethod`, we can rename and modify it to work with known and unknown methods as well.